pkg/apis/marketplace/v1alpha1/operatorsource_types.go

```diff
@@ -49,6 +49,12 @@ type OperatorSourceSpec struct {
 	// manifests under this namespace will be visible.
 	// Please note that this is not a k8s namespace.
 	RegistryNamespace string `json:"registryNamespace,omitempty"`
+
+	// GroupName specifies the group name of the operator offering(s) associated
+	// with given OperatorSource object. For example, an admin can specify
+	// "Community" or "ISV" as the group name.
+	// This is an optional attribute.
+	GroupName string `json:"group,omitempty"`
```

Comment:
The original jira story says by `Operator Type` which is not clear. Please suggest a better name.

Reply:
Shouldn't it be based on whatever the field looks like on the UI? I assume that field will be visible there.

Reply:
Do we really need to expose this to the end user? Can't we just add labels to the `OperatorSource` CR and then pass them down to the `CatalogSourceConfig` and `CatalogSource`? This way if the UI filtering requirements keep growing it can be handled without making any operator changes.  

Moreover I think these labels will be used only in the default CRs that we are adding to the cluster. I don't believe we want expose this to the cluster admin. Yes they could add labels to their CRs but at least we are not advertising it at the `OpetatorSource` API level. We can then revisit UI requirements in the 4.1 time-frame and introduce this if needed at that point.